core/input/mouse.js

```diff
@@ -12,12 +12,20 @@ import * as Log from '../util/logging.js';
 import { isTouchDevice } from '../util/browsers.js';
 import { setCapture, stopEvent, getPointerEvent } from '../util/events.js';
 import { set_defaults, make_properties } from '../util/properties.js';
+import { getScrollLineHeight } from "./util.js";
 
 function Mouse(defaults) {
 
     this._doubleClickTimer = null;
     this._lastTouchPos = null;
 
+    this._wheelStepTimer = null;
+    this._accumulatedWheelDeltaX = 0;
+    this._accumulatedWheelDeltaY = 0;
+
+    this.WHEEL_STEP = 10; // Delta threshold for a mouse wheel step
+    this.WHEEL_LINE_HEIGHT = getScrollLineHeight();
```

Comment:
These are constants I assume? So why not put them at the top level?

Reply:
Yes. You mean outside the mouse object? I'll move them